web-registry/src/components/organisms/SellOrdersActionsBar/SellOrdersActionsBar.tsx

```diff
@@ -16,46 +18,71 @@ type Params = {
   isBuyButtonDisabled: boolean;
   onSellButtonClick: () => void;
   onBuyButtonClick: () => void;
+  onChainProjectId?: string;
+  projectName?: string;
+  onChainCreditClassId?: string;
+  creditClassName?: string;
```

Comment:
i added these fields so that we can send this as buy1 metadata for the sticky nav. i.e. if the user clicks buy in the sticky nav from a project page we would send projectId and projectName. and a similar thing happens if buy is clicked from the credit class page. we send creditClassId and creditClassName.

Reply:
what about just getting rid of the `onChain` prefix here?

Reply:
the reason i chose those names here is because they are named this way in the parent components. i'd rather keep it consistent with the names in the parent components or do the rename all the way up through the parent components. i'm preferring to just keep it as is right now.

that said, i have spoken with @mhagel about the names of these things, particularly in the project page details, and we had some agreement that the naming of things was confusing and probably worth refactoring. maybe we can create a separate issue now to address that later? WDYT @blushi ?

Reply:
sounds good